src/api/db/models/Camera.ts

```diff
@@ -236,6 +237,72 @@ export class CameraModel {
     }
   }
 
+  static async removeProjectRegistration(
+    input: { cameraId: string },
+    context: Pick<Context, 'user'>,
+  ): Promise<{ wirelessCameras: WirelessCameraSchema[]; project?: ProjectSchema }> {
+    const projectId = context.user['curr_project'];
+
+    try {
+      const wirelessCameras = await WirelessCamera.find();
+      const cam = wirelessCameras.find((c) => idMatch(c._id, input.cameraId));
+
+      if (!cam) {
+        const msg = `Couldn't find camera record for camera ${input.cameraId}`;
+        throw new CameraRegistrationError(msg);
+      }
+      let addedNewCamConfig = false;
+      let defaultProj: Maybe<ProjectSchema> = null;
+
+      const activeReg = cam.projRegistrations.find((pr) => pr.active);
+
+      // if active registration === curr_project,
+      // set default_project registration to active
+      if (activeReg?.projectId === projectId) {
+        const defaultProjReg = cam.projRegistrations.find(
+          (pr) => pr.projectId === 'default_project',
+        );
+        if (defaultProjReg) {
+          defaultProjReg.active = true;
+        } else {
+          // make sure there's a Project.cameraConfig record for this camera
+          // in the default_project and create one if not
+          defaultProj = await Project.findOne({ _id: 'default_project' });
```

Comment:
@nathanielrindlaub slightly different approach to this than the `unregisterCamera` method above, but I make sure `default_project` exists first and then create a `CameraConfig` in the `default_project` project before creating the project registration. This way we make sure the project exists first, instead of having a camera project registration point to a non-existant project, but also not sure if this is even an issue that needs to be addressed.

Reply:
@jue-henry I don't think there not being a `default_project` in the DB should be an issue. It gets [created](https://github.com/tnc-ca-geo/animl-api/blob/d0da6f81805e2721a1c2bc7e7df6d721c36f4f88/src/scripts/seedDb.js#L170C1-L184C5) when we initially seed the DB in the `src/scripts/seedDB.js` script. It is conceivable that someone might standing up their own Animl instance and not run the script, but that would cause a lot of other issues long before anyone every got to the point of deleting a camera, so I think it's safe to assume that `default_project` exists.